src/time_zone_info.cc

```diff
@@ -361,8 +361,10 @@ bool TimeZoneInfo::ExtendTransitions() {
   std::int_fast64_t jan1_time = jan1 - civil_second();
   int jan1_weekday = ToPosixWeekday(get_weekday(jan1));
 
-  Transition dst = {0, dst_ti, civil_second(), civil_second()};
-  Transition std = {0, std_ti, civil_second(), civil_second()};
+  Transition dst = {0, static_cast<uint_least8_t>(dst_ti), civil_second(),
+                    civil_second()};
+  Transition std = {0, static_cast<uint_least8_t>(std_ti), civil_second(),
+                    civil_second()};
```

Comment:
I'll probably change this so that `dst_ti`/`std_ti` are just `uint_least8_t`s in the first place, but I don't hold you up now.